### PR TITLE
make https scheme to constant

### PR DESCRIPTION
**Pull request by xiqiu618**
### Description of your changes
make https scheme to constant

Fixes #

I have:

- [x] Read and followed Crossplane's [contribution process].
- [x] Run `make reviewable test` to ensure this PR is ready for review.

### How has this code been tested

[contribution process]: https://git.io/fj2m9

### CODE REVIEW

**Comment by xiqiu618** [comment]
@wonderflow  @negz  please help me approve

File: pkg/clients/rds/rds.go [hunk 1]
```diff
@@ -36,6 +36,10 @@ var (
 	ErrCodeInstanceNotFound = "InvalidDBInstanceId.NotFound"
 )
 
+const (
+	httpsScheme = "https"
```
Comment (zzxwill) [inline pkg/clients/rds/rds.go:40]:
```suggestion
	httpsScheme = "Https"
```

**Review by zzxwill — APPROVED** [review]
/gtm

File: pkg/clients/rds/rds.go (hunk 1 above)
Comment (xiqiu618) [reply pkg/clients/rds/rds.go:40]: internal constant, exposure may bring risk

**Review by wonderflow — APPROVED** [review]

**Pull request merged**